setup.py

```diff
@@ -31,30 +31,31 @@
 
 kwargs = {
     'name': 'rocker',
-    'version': '0.2.3',
+    'version': '0.3.0',
     'packages': ['rocker'],
     'package_dir': {'': 'src'},
     'package_data': {'rocker': ['templates/*.em']},
     'entry_points': {
         'console_scripts': [
             'rocker = rocker.cli:main',
             'detect_docker_image_os = rocker.cli:detect_image_os',
-	    ],
+        ],
```

Comment:
please avoid unrelated changes to the PR.

Reply:
A little cleanup can be done, but it should be at least separated to a different commit.